### DISCUSSION TITLE
Is it possible to only request "read" permissions for Intune Manager and exclude "write" permissions in the request?

### DISCUSSION THREAD

**Discussion by surferstylee** [Micke-K/IntuneManagement#251] [General]
I have a customer who was double hacked in the same week and is still shell-shocked and weary of the "write" permissions in the request I have submitted. Since I am only using the tool to document their environment, is there a way to "easily" change the request to exclude any "write permissions or is this a heavier task than I want to know?

![image](https://github.com/Micke-K/IntuneManagement/assets/17766354/a942a106-a136-45d2-8791-cf4c5a755502)

**Comment by Micke-K** [comment]
Hello,

There is no option for this. You would have to add permissions manually.

One quick and dirty way to do this is to update all .ReadWrite. in the EndpointManager.psm1 file to .Read.
A simple search and replace would fix that.

Note that I have not tested this myself, but this should in theory fix it.

Cheers!

**Comment by surferstylee** [comment]
Thank you very much and I'll give the find and replace option a great run
in my lab and then with my customer.

I'll update you after I get some data/results my friend!

Cheers m8! 🤙

~m
Cheers,  🤙

~m
